ppr-ui/src/components/common/SimpleHelpToggle.vue

```diff
@@ -10,7 +10,9 @@
       <v-icon class="mr-1">
         mdi-help-circle-outline
       </v-icon>
-      {{ isHelpContentOpen ? 'Hide ' + title : title }}
+      {{ isHelpContentOpen ?
+          defaultHideText ? 'Hide Help' : 'Hide ' + title
+          : title }}
```

Comment:
I think this could be simplified.
Maybe a simple computed value vs nested ternary? The latter is usually something we want to avoid, can be hard to read.

Reply:
Agreed thanks for the suggestion!